cmd/activator/main.go

```diff
@@ -133,6 +152,73 @@ func main() {
 		logger.Fatalw("Failed to create stats reporter", zap.Error(err))
 	}
 
+	// Set up signals so we handle the first shutdown signal gracefully.
+	stopCh := signals.SetupSignalHandler()
+
+	kubeInformerFactory := kubeinformers.NewSharedInformerFactory(kubeClient, defaultResyncInterval)
+	servingInformerFactory := servinginformers.NewSharedInformerFactory(servingClient, defaultResyncInterval)
+	endpointInformer := kubeInformerFactory.Core().V1().Endpoints()
+	revisionInformer := servingInformerFactory.Serving().V1alpha1().Revisions()
+
+	go revisionInformer.Informer().Run(stopCh)
+	go endpointInformer.Informer().Run(stopCh)
```

Comment:
I think you removed the wrong lines :)

These two should start the informer factories, which don't need `go` in front.

Reply:
As per slack discussion, I added a comment about that. 
In short, I intentionally Run the informers instead of starting them from the factory. Otherwise, they are hanging waiting for a sync.